### PR TITLE
feat(cookie): add atom for manipulate cookie based on @cookie-baker

### LINKED ISSUE #388: Add atom for manipulate cookie
Based on lib [cookie-baker](https://github.com/BataevDaniil/cookie-baker)

### PR DESCRIPTION
**Pull request by BataevDaniil**

### CODE REVIEW

File: packages/cookie/src/cookie.test.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  }
+  const cookieAtom = createCookieAtom(cookie, realTimeCookie)
+  let result = null
+  cookieAtom.subscribe((x) => (result = x))
+  handler(newCookie)
+  assert.equal(actual, result)
+})
+
+// TODO
+test.skip('unsubscribe from RealTimeCookie when have not subscriber', () => {
```
Comment (BataevDaniil) [inline packages/cookie/src/cookie.test.ts]: We will have an api for unsubscribing a listener when there are no subscribers? @artalar

File: packages/cookie/src/cookie.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { action, atom } from '@reatom/core'
+import {
+  CookieController,
+  CookieObjectModel,
+  RealTimeCookie,
+} from '@cookie-baker/core'
+import { onConnect, withInit } from '@reatom/hooks'
+
+export const createCookieAtom = <T extends CookieObjectModel>(
```
Comment (artalar) [inline packages/cookie/src/cookie.ts]:
```suggestion
export const reatomCookie = <T extends CookieObjectModel>(
```

File: packages/cookie/src/cookie.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  cookie: CookieController<T>,
+  realTimeCookie: RealTimeCookie<T>,
+) => {
+  const cookieAtom = atom({} as ReturnType<typeof cookie.get>).pipe(
+    withInit(() => cookie.get()),
+  )
+  let updateCookie: Parameters<typeof realTimeCookie.addListener>[0]
+  onConnect(cookieAtom, (ctx) => {
+    ctx.schedule(() => {
```
Comment (artalar) [inline packages/cookie/src/cookie.ts]: `onConnect` is already an effect, you don't need a `schedule`

File: packages/cookie/src/cookie.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      updateCookie = (newCookie) => cookieAtom(ctx, newCookie)
+      realTimeCookie.addListener(updateCookie)
+    })
+    return () => {
+      ctx.schedule(() => {
```
Comment (artalar) [inline packages/cookie/src/cookie.ts]: Same, `schedule` isn't needed

**Pull request merged**